protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -146,6 +146,32 @@ An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the s
 (average entry valuation) = 1090.9
 ```
 
+### Calculating supplied liquidity from probability of trading 
+
+At every vega time change calculate the supplied liquidity provided by each committed LP. 
+This is done by taking into account all the volume they're providing between the tightest price monitoring bound and then 
+use the formula provided by [probability weighted liquidity measure](./0034-PROB-prob_weighted_liquidity_measure.ipynb).
+
+Now calculate the total provided liquidity by committed LPs:
+```
+total_provided = sum over committed LPs of liquidity provided by a committed LP
+```
+
+Now calculate fraction of liquidity provided for a committed LP:
+```
+fraction of liquidity provided by committed LP = liquidity provided by a committed LP / total_provided
+```
```

Comment:
```suggestion
```
If the `total_provided` comes out as `0` then set `fraction of liquidity provided by committed LP` to `1.0` for all committed LPs. 
```